src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1098,3 +1099,99 @@ export function getProfileWithBalancedNativeAllocations(): * {
 
   return { profile, funcNamesDict };
 }
+
+export function addActiveTabInformationToProfile(
+  profile: Profile,
+  activeBrowsingContextID?: BrowsingContextID
+): * {
+  const firstTabBrowsingContextID = 1;
+  const secondTabBrowsingContextID = 4;
+  const parentInnerWindowIDsWithChildren = 11111111111;
+  const iframeInnerWindowIDsWithChild = 11111111112;
+  const fistTabInnerWindowIDs = [
+    parentInnerWindowIDsWithChildren,
+    iframeInnerWindowIDsWithChild,
+    11111111113,
+    11111111115,
+  ];
+  const secondTabInnerWindowIDs = [11111111114, 11111111116];
+
+  // Default to first tab browsingContextID if not given
+  activeBrowsingContextID =
+    activeBrowsingContextID === undefined
+      ? firstTabBrowsingContextID
+      : activeBrowsingContextID;
+
+  // Add a pages array with the following relationship:
```

Comment:
This might be nice to have in function documentation at the top, so you don't have to read the implementation. Also, nice docs!